cli/cmd/run.go

```diff
@@ -101,20 +84,30 @@ func internalRunModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 		return errNoConfig
 	}
 
-	runOpts := newRunOpts(*cmdCtx)
+	runInfo := newRunInfo(*cmdCtx)
 	scriptPath := ""
 	startIndex := 0
 	if len(args) > 0 {
-		if strings.HasSuffix(args[0], ".lua") {
-			scriptPath = args[0]
-			if _, err := os.Stat(scriptPath); err != nil {
-				return fmt.Errorf("there was some problem locating script: %s", err)
+		for i, option := range args {
+			if strings.HasSuffix(option, ".lua") {
```

Comment:
Yes, i think so. Because if there is a script, we launch it using our wrapper.
[here](https://github.com/tarantool/tt/blob/550b7b04f996d10a3337a08ccef3f64ca9e870c6/cli/running/script_instance.go#L205)

Reply:
Wow... You know why do we need launcher for `tt run`? I think this is an error.

Reply:
> Wow... You know why do we need launcher for `tt run`? I think this is an error. @LeonidVas what do you think?

AFAIR that was the point of 'tt run' to run instances using our wrapper from the beginning,

Reply:
That is why I'm asking :) 
Not sure this is very useful feature without all the  TT_* variables set, which is done by `tt`. 
But it brings more complexity to current run implementation: we need to be more careful with reading/parsing CLI args. For example, if we want to search for .lua script names manually in command line to support launcher script, we need to process different scenarios correctly:
`$ tt run -e my.lua`
This is not `my.lua` script invocation, but current implementation will search for `my.lua` script.

Reply:
We have discussed this with @LeonidVas and decided to move towards `tt run` = `tarantool` without any hidden script invocation. So looks like we can remove script name searching in argv and `launcher.lua` injection for `tt run`.